
string tagAddress
float64 rangeNum
float64 timeOfReception
float64 distance
float64 degrees
float64 Xcoord
float64 Ycoord
float64 clockOffset
float64 serviceData

